airflow/metrics/validators.py

```diff
@@ -118,7 +106,7 @@ def get_validator() -> ListValidator:
     else:
         list_type = DEFAULT_VALIDATOR_TYPE
 
-    return validators[validator_type][list_type](metric_lists[list_type])
+    return validators["pattern"][list_type](metric_lists[list_type])
```

Comment:
If the above is accepted then also:

```suggestion
    return validators[list_type](metric_lists[list_type])
```

Reply:
done